build.gradle.kts

```diff
@@ -86,3 +86,13 @@ apiValidation {
 
   ignoredProjects.addAll(ignoreApiValidation)
 }
+
+rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.yarn.YarnPlugin> {
```

Comment:
🙏

Reply:
Unfortunately it seems that blindly updating the versions breaks the build 😞 So I reverted it, I guess we'll have to live with Dependabot alerts...

Reply:
I guess it was not related to this. Resolving the comment